l10n.yaml

```
@@ -6,3 +6,4 @@ preferred-supported-locales: [ru]
 synthetic-package: false
 nullable-getter: false
 format: true
+untranslated-messages-file: untranslatedmessage.txt
```

Apply suggestion and and this file into .gitignore

```suggestion
untranslated-messages-file:  lib/src/shared/l10n/untraslated_keys.json
```